schemas/v3.1/schema.json

```diff
@@ -277,7 +277,7 @@
     "paths": {
       "type": "object",
       "patternProperties": {
-        "^/": {
+        "^/[^#?]*$": {
```

Comment:
As mentioned on Slack, I believe this restriction would be going beyond what the markdown source-of-truth dictates (solely that the path key MUST start with a forward slash). Though unescaped `#` and '?' characters cannot appear within path template variables, there is afaik no restriction on them occurring **outside** those parameters. I make no comment on whether that is fortunate or not.